aea/agent.py

```diff
@@ -104,7 +104,7 @@ def __init__(
 
         self._multiplexer = Multiplexer(self._connections, loop=loop)
         self._inbox = InBox(self._multiplexer)
-        self._outbox = OutBox(self._multiplexer)
+        self._outbox = OutBox(self._multiplexer, identity.address)
```

Comment:
We provide the default address to the outbox to simplify the `put_message` api (unless a non-default address is used; the `sender` field can be populated framework side)